app/views/pages/companyDetails/businessContactDetails.scala.html

```diff
@@ -17,54 +17,56 @@
 @import helpers.templates.{oneOfManyErrorWrapper, payeErrorSummary, payeInput}
 @import models.DigitalContactDetails
 @import uk.gov.hmrc.play.views.html.{helpers => govHelpers}
-@(contactForm: Form[DigitalContactDetails])(implicit request: Request[_], messages: Messages)
+@(contactForm: Form[DigitalContactDetails], companyName: String)(implicit request: Request[_], messages: Messages)
+
+
 
 @formContent = {
     <fieldset>
         <legend class="visually-hidden">
-            @Messages("pages.businessContact.description")
+        @Messages("pages.businessContact.description")
         </legend>
         <div class="form-group">
-            @payeInput(
+        @payeInput(
```

Comment:
Reformatting is a little odd here, IntelliJs twirl formatting isn't great.

I'd maybe be more inclined to indent beneath the div to show everything contained within the div.